cpu/esp_common/Makefile.include

```diff
@@ -105,3 +105,45 @@ PROGRAMMER ?= esptool
 
 # Add esptool in the list supported programmers
 PROGRAMMERS_SUPPORTED += esptool
+
+# The ELFFILE is defined by default in $(RIOTBASE)/Makefile.include but only
+# after the this file is included, so we need ELFFILE defined earlier.
+# This is used to create new make rules in this file (based on FLASHFILE)
+# and can't be deferred.
+ELFFILE ?= $(BINDIR)/$(APPLICATION).elf
+FLASHFILE ?= $(ELFFILE).bin
```

Comment:
Does this comment still apply?

Reply:
Yes. Without ELFFILE defined here I get: 
```
$ make
Building application "paho-mqtt-example" for "esp32-ethernet-kit-v1_2" with MCU "esp32".

make: *** No rule to make target '.../esp/RIOT/examples/paho-mqtt/bin/esp32-ethernet-kit-v1_2/paho-mqtt-example.elf.bin', needed by 'link'.  Stop.
```

Reply:
Without `FLASHFILE` I get

```
$ make
.../esp/RIOT/examples/paho-mqtt/../../Makefile.include:634: *** FLASHFILE is not defined for this board: .  Stop.
```